### ISSUE TITLE
Python container

### ISSUE THREAD

**Issue by CustomIcon** [uwu/basic-env#3]
is it possible to build python container with base image.
Id love to see it happen officially

**Comment by Nandini1071** [comment]
Hi @CustomIcon is this issue still open, if yes then what is the requirement would like to contribute on this.

**Comment by CustomIcon** [comment]
> Hi @CustomIcon is this issue still open, if yes then what is the requirement would like to contribute on this.

Would love to have pyenv project image here
source: https://github.com/pyenv/pyenv

its a python version manager

also in addition instead of nodejs can we have nvm?
source: https://github.com/nvm-sh/nvm

different people like to have different runtime versions for different projects. would love to see it happen

**Comment by Nandini1071** [comment]
> > Hi @CustomIcon is this issue still open, if yes then what is the requirement would like to contribute on this.
> 
> Would love to have pyenv project image here source: https://github.com/pyenv/pyenv
> 
> its a python version manager
> 
> also in addition instead of nodejs can we have nvm? source: https://github.com/nvm-sh/nvm
> 
> different people like to have different runtime versions for different projects. would love to see it happen

For the first one image which image needs to be added and in which folder and tech stack also please mention??

**Comment by CustomIcon** [comment]
> > > Hi @CustomIcon is this issue still open, if yes then what is the requirement would like to contribute on this.
> 
> > 
> 
> > Would love to have pyenv project image here source: https://github.com/pyenv/pyenv
> 
> > 
> 
> > its a python version manager
> 
> > 
> 
> > also in addition instead of nodejs can we have nvm? source: https://github.com/nvm-sh/nvm
> 
> > 
> 
> > different people like to have different runtime versions for different projects. would love to see it happen
> 
> 
> 
> For the first one image which image needs to be added and in which folder and tech stack also please mention??

currently im just using docker env with the image vicamo/pyenv:slim-buster
![image](https://github.com/user-attachments/assets/c84f1f1d-ea6b-46e4-a2da-17a000aa7d74)

and modified startup script that adds PYENV_ROOT to the PATH as follows:
![image](https://github.com/user-attachments/assets/0f7e1acb-97a6-4761-821c-fd73dcd43110)

**Comment by Nandini1071** [comment]
Actually I am not familiar with docker but proficient in python so anything on which I can contribute on python side let me know would like to help on this side

**Comment by CustomIcon** [comment]
> Actually I am not familiar with docker but proficient in python so anything on which I can contribute on python side let me know would like to help on this side

well actually, we have about 3 images that is working:
ghcr.io/uwu/basic-env/dart:latest
ghcr.io/uwu/basic-env/javascript:latest
ghcr.io/uwu/basic-env/java:latest

so why dont we have ghcr.io/uwu/basic-env/python:latest using base image

**Comment by Nandini1071** [comment]
yeah that's what not sure about images and docker but would be able to contribute on python code, let me know if you need any help on that side. @CustomIcon.
